TDD_Training/FindMultiplesTests/FindMultiplesTests.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using TDD_Training;
+using Xunit;
+
+namespace FindMultiplesTests
+{
+    public class FindMultiplesTests
+    {
+        private readonly FindMultiples findMultiplesService = new FindMultiples();
+
+        [Fact]
+        public void CheckGivenNumber_IfMultipleOfThree_ReturnsFizz()
```

Comment:
Why not move this also to the below Theory Data?

Reply:
I wrote one test based on [Fact] and rest on [theory]. Just to have an example for both

Reply:
Yep. That's good. But you can merge this along with Theory Data